config/samples/xgboost-dist/README.md

```diff
@@ -21,18 +21,19 @@ docker push kubeflow/xgboost-dist-iris-test:1.0 ./
 
 **Configure the job runtime via Yaml file**
 
-There are two yaml files to setup distributed XGBoost computation runtime. 
-For training job, you could configure xgboostjob_v1alpha1_iris_predict.yaml. 
-Note, we use [OSS](https://www.alibabacloud.com/product/oss) to store the trained model, 
-thus, you need to specify the OSS parameter in the yaml file. Therefore, remember to fill the OSS parameter in the yaml file. 
-The oss parameter includes the account information such as access_id, access_key, access_bucket and endpoint. 
+There are four yaml files to setup distributed XGBoost computation runtime. they are : xgboostjob_v1alpha1_iris_train.yaml, xgboostjob_v1alpha1_iris_train_local.yaml, xgboostjob_v1alpha1_iris_predict.yaml, xgboostjob_v1alpha1_iris_predict_local.yaml.
```

Comment:
Make the file list bulleted with a sub-heading for OSS and Local

Reply:
done